### PR TITLE
[CI] Enable no-body checker in Github Action

### PR DESCRIPTION
**Pull request by llFreetimell**
This commit enables no-body checker in Github Action.

ONE-vscode-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

---

Parent Issue : #123 
Draft : #308

### CODE REVIEW

File: .github/workflows/check-commit.yml [hunk 1]
```diff
@@ -28,3 +28,28 @@ jobs:
         run: |
           cat signed_off_by_checker.fail
           exit 1
+
+  check-no-doby:
```
Comment (jyoungyun) [inline .github/workflows/check-commit.yml:32]:
```suggestion
  check-no-body:
```

**Review by YongseopKim — DISMISSED** [review]
LGTM

**Comment by llFreetimell** [comment]
@YongseopKim Sorry, bugfix was not applied at first :( PTAL!

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by YongseopKim — APPROVED** [review]

**Pull request merged**